test/expected/cluster-14.out

```diff
@@ -22,9 +22,7 @@ INSERT INTO cluster_test VALUES ('2017-01-20T09:00:01', 23.4, 1),
 -- Run cluster
 CLUSTER VERBOSE cluster_test USING cluster_test_time_idx;
 INFO:  clustering "_timescaledb_internal._hyper_1_1_chunk" using index scan on "_hyper_1_1_chunk_cluster_test_time_idx"
-INFO:  "_hyper_1_1_chunk": found 0 removable, 1 nonremovable row versions in 1 pages
 INFO:  clustering "_timescaledb_internal._hyper_1_2_chunk" using index scan on "_hyper_1_2_chunk_cluster_test_time_idx"
-INFO:  "_hyper_1_2_chunk": found 0 removable, 1 nonremovable row versions in 1 pages
```

Comment:
Here is seems reasonable to show INFO level events since it verifies that clustering actually execute.